content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,656 @@
+
+1. **Kubernetes (K8s) only:**  Istio in ambient mode is currently only supported for deployment on Kubernetes clusters. Deployment on non-Kubernetes endpoints such as virtual machines is not currently supported.
+
+1. **No Istio Multi-cluster support:**  Only single cluster deployments are currently supported for Istio Ambient mode.
+
+1. **K8s CNI restrictions:**  Istio in Ambient mode does not currently work alongside every single Kubernetes CNI plugin implementations Additionally, with some plugins. certain CNI functions (in particular Kubernetes NetworkPolicy and Kubernetes Service Load balancing features) may get transparently bypassed in the presence of Istio Ambient mode. The exact set of supported CNI plugins as well as any CNI feature caveats are currently under test and will be formally documented as Istio's ambient mode approaches the beta release.
+ 
+1. **TCP/ IPv4 only:** In the current release, TCP over IPv4 is the only protocol supported for transport on an Istio HBONE tunnel (this includes protocols such as HTTP that run between application layer endpoints on top of the TCP/ IPv4 combination independent of the outermost HTTP used for the HBONE transport between ztunnel proxies).
+
+1. **No dynamic switching to ambient mode:** Ambient mode can only be enabled on a new Istio mesh control plane that is deployed using ambient profile or helm configuration. An existing Istio mesh deployed using a pre-ambient profile for instance can not be dynamically switched to also enable ambient mode operation.  
+
+1. **Restrictions with Istio PeerAuthentication:**  as of the time of writing, the PeerAuthentication resource  is not supported by all components (i.e. Waypoint proxies) in Istio ambient mode. Hence it is recommended to only use the STRICT mTLS mode currently, or simply use default Istio settings which automatically enable mTLS on all endpoints. Like many of the other alpha stage caveats, this shall be addressed as the feature moves toward beta status. 
+
+1. **Istioctl CLI gaps:** There may be some minor functional gaps in areas such as istioctl CLI output displays when it comes to displaying or monitoring Istio's ambient mode related information. These will be addressed as the feature matures.
+
+### Environment used for this guide 
+
+For the examples in this guide, we used a deployment of Istio version 1.19.0 on a `kind` cluster of version 0.20.0 running Kubernetes version 1.27.3. However these should also work on any Kubernetes cluster at version 1.24.0 or later and Istio version 1.18.0 or later.  It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. Refer to the [installation user guide](/docs/ops/ambient/usage/install/) or [Getting started guide](/docs/ops/ambient/getting-started/) information on installing Istio in ambient mode on a Kubernetes cluster. 
```

Comment:
Changed this to ...

For the examples in this guide, we used a deployment of Istio version 1.19.0 on a `kind` cluster of version 0.20.0 running Kubernetes version 1.27.3. The minimum Istio version needed for ambient functions is 1.18.0 and the minimum Kubernetes version needed is 1.24.0.  It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. Refer to the [installation user guide](/docs/ops/ambient/usage/install/) or [getting started guide](/docs/ops/ambient/getting-started/) for information on installing Istio in ambient mode on a Kubernetes cluster.